lab-justin/src/__test__/note-create-form.test.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+import NoteCreateForm from '../component/note-create-form';
+
+describe('testing NoteCreateForm', () => {
```

Comment:
These tests are not really testing what you want them to. You should check out enzyme's [`simulate`](http://airbnb.io/enzyme/docs/api/ShallowWrapper/simulate.html) function.